l10n_br_fiscal/models/operation_line.py

```diff
@@ -197,6 +206,9 @@ def map_fiscal_taxes(self, company, partner, product=None,
         cfop = self._get_cfop(company, partner)
         mapping_result['cfop'] = cfop
 
+        if tax_calc == TAX_CALC_ONLY:
```

Comment:
Se esse método for chamado com o TAX_CALC_MANUAL vai calcular automaticamente os impostos... o correto seria fazer if tax_calc != TAX_CALC_AUTO